assets/javascripts/discourse-assign/connectors/advanced-search-options-above/assigned-advanced-search.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="control-group pull-left">
```

Comment:
Can we move this to the advanced-search-options**-below** outlet? I don't think it should be right at the top, since it is for a plugin.